send-transaction-service/src/send_transaction_service.rs

```diff
@@ -311,14 +302,26 @@ impl SendTransactionService {
         result
     }
 
-    fn send_transaction(
-        send_socket: &UdpSocket,
-        tpu_address: &SocketAddr,
-        wire_transaction: &[u8],
-    ) {
-        if let Err(err) = send_socket.send_to(wire_transaction, tpu_address) {
```

Comment:
we need to retain the ability to optionally send via UDP

Reply:
hmm, the plan is to have the connection_cache to do that based on configuration. It will have the option to that. I will take a crack at it.